src/OpenSage.Game/Gui/Apt/RenderItem.cs

```diff
@@ -1,29 +1,26 @@
 ﻿using OpenSage.Data.Apt.Characters;
 using OpenSage.Gui.Apt.ActionScript;
+using Veldrid;
 
 namespace OpenSage.Gui.Apt
 {
     public sealed class RenderItem : IDisplayItem
     {
-        private SpriteItem _parent;
-        private Character _character;
-        private AptContext _context;
-        private ObjectContext _scriptObject;
-
-        public SpriteItem Parent => _parent;
-        public Character Character => _character;
-        public AptContext Context => _context;
+        public SpriteItem Parent { get; private set; }
```

Comment:
I think all these private setters can be removed entirely.

Reply:
Doesn't work for me when removing them. I guess i can only set them from the constructor without the `private set`